x-pack/plugins/siem/public/graphql/types.ts

```diff
@@ -5405,6 +5405,8 @@ export namespace GetOneTimeline {
 
     pinnedEventsSaveObject: Maybe<PinnedEventsSaveObject[]>;
 
+    status: Maybe<TimelineStatus>;
+
```

Comment:
Something changed in the graphQL generator and/or the GraphQL. I verified this on master and appreciate you adding the extra fix here as well 👍

Reply:
Yea, Patrick is quickly adding it in here: https://github.com/elastic/kibana/pull/68137 .